geode-gfsh/src/test/java/org/apache/geode/management/internal/cli/commands/CreateGatewaySenderCommandTest.java

```diff
@@ -123,6 +123,22 @@ public void whenCommandOnMember() {
         .statusIsSuccess();
   }
 
+  @Test
+  public void testSingleDispatcherThread() {
+    doReturn(mock(Set.class)).when(command).getMembers(any(), any());
+    cliFunctionResult = new CliFunctionResult("member",
+        CliFunctionResult.StatusState.OK, "cliFunctionResult");
+    functionResults.add(cliFunctionResult);
+    gfsh.executeAndAssertThat(command,
+        "create gateway-sender --member=xyz --id=1 --remote-distributed-system-id=1 " +
```

Comment:
If there is no test with null or invalid order policy; please add one.

Reply:
Test added.